nitro-protocol/.eslintignore

```diff
@@ -0,0 +1 @@
+dist
```

Comment:
```suggestion
dist

```

Reply:
Why?

Reply:
Just convention, but it's "polite" to end files with newlines so that they (for example) behave well with `cat`.

Reply:
(The diff viewer for this PR flagged that the file didn't end with a newline)